### ISSUE TITLE
Feature Request: 2 Factor Chatbot

### ISSUE THREAD

**Issue by parksddd** [keybase/client#21293]
Probably far outside of scope, but using keybase similar to Okta/Symantec VIP/Duo/Ping would be a pretty nifty feature. Probably would need to limit the interaction to specific device(s).

Login Form (Username) > Keybase Message > Reply/Click > Authenticated

**Comment by maxtaco** [comment]
We've brainstormed this inhouse. It should be very doable, you're encouraged to try it!

**Comment by xgess** [comment]
i agree! here's an example of a TOTP bot (quite a bit less slick than the flow you've described) we recently wrote if you're curious: https://github.com/keybase/pykeybasebot/blob/master/examples/4_totp_storage.py
